features/fixtures/shared/scenarios/NetworkBreadcrumbsScenario.swift

```diff
@@ -10,18 +10,12 @@ import Foundation
 
 @available(iOS 10.0, macOS 10.12, *)
 class NetworkBreadcrumbsScenario : Scenario {
-    
-    lazy var baseURL: URL = {
-        var components = URLComponents(string: Scenario.mazeRunnerURL.absoluteString)!
-        components.port = 9340 // `/reflect` listens on a different port :-((
-        return components.url!
-    }()
-    
+        
```

Comment:
I added the `/reflect` endpoint to the server that runs on 9339 to prevent us from having to do this.